fix: document only using 1 ske-gen per workflow (#105)
syntasso/kratix-docs: docs/ske/10-integrations/10-backstage/03-generator.mdx

## How does it work?

The `ske-backstage-generator` image is intended to be run as part of your
Configure Pipeline for your Promise and Resource Workflows. It will
auto-generate Backstage Template and Component manifests, and schedule them to
your [Backstage Destination](#destination). **You must ensure this is only run

> once across all your pipelines** in a workflow. In order to
> have the full status displayed in Backstage, you should place the aspect as the

Reviewer: ```suggestion
once across all your pipelines** in a workflow. 

In order to
have the full status displayed in Backstage, you should place the aspect as the
```

Maybe a single new line to break up the text?

> last image in the last pipeline of your workflow. If you have this more than
> once in your workflow, you will see a random one of the components in Backstage,

Reviewer: ```suggestion
last image in the last pipeline of your workflow. If the aspect appears more than
once in your workflow, you will see one of the components in Backstage,
```
